packages/devtools_app_shared/lib/src/service/eval_on_dart_library.dart

```diff
@@ -41,11 +41,16 @@ class EvalOnDartLibrary extends DisposableController
     ValueListenable<IsolateRef?>? isolate,
     this.disableBreakpoints = true,
     this.oneRequestAtATime = false,
+    this.forceMainIsolate = false,
```

Comment:
instead of adding a new parameter, can we just pass `serviceManager.isolateManager.mainIsolate` to the existing `isolate` parameter?